chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/listing/pending/PendingCertifiedProductEntity.java

```diff
@@ -204,7 +204,8 @@
     @Column(name = "pending_certified_product_id", nullable = false)
     private Set<PendingCertifiedProductParentListingEntity> parentListings;
     
-    @Transient
+    @Basic(optional = false)
+    @Column(name = "has_qms", nullable = false)
     private boolean hasQms;
```

Comment:
There should be a data model PR to match this - where is it? It's why I suggested you push your data model changes to Jenkins via a new job b/c your build was failing due to missing this column.